dataactcore/interfaces/function_bag.py

```diff
@@ -350,4 +350,23 @@ def getErrorMetricsByJobId(job_id, include_file_types=False, severity_id=None):
             record_dict['source_file'] = FILE_TYPE_DICT_ID.get(result.file_type_id, '')
             record_dict['target_file'] = FILE_TYPE_DICT_ID.get(result.target_file_type_id, '')
         result_list.append(record_dict)
-    return result_list
\ No newline at end of file
+    return result_list
+
+def get_cross_file_combos():
+    """Return a list of source and target file types used in cross-file validations."""
+    sess = GlobalDB.db().session
+
+    # note: we query the file_type table here instead of
+    # using FILE_TYPE_DICT in lookups.py because cross-file
+    # validations don't apply to all file types (file types
+    # with a NULL file_order--like E and F--are excluded
+    # from the results of the query below)
+    target_files = sess.query(FileType).subquery()
+    cross_file_combos = sess.query(
+        FileType.name.label('first_file_name'),
+        FileType.file_type_id.label('first_file_type_id'),
+        target_files.c.name.label('second_file_name'),
+        target_files.c.file_type_id.label('second_file_type_id')
+    ).filter(FileType.file_order < target_files.c.file_order)
```

Comment:
Outside the scope of this PR, but I wanted to see what you'd think:
1. Hitting the DB to rely on some implicit NULL handling feels a little icky. Would it make sense to do the logic (including excluding E and F) in Python?
2. I _did_ like that the previous version dealt with pairs of objects (though `source` and `target` weren't great names) as opposed to a sort of joined object with field names prefixed with `first` and `second`. Which do you think is easier to work with?